RationaleMCP/0031/annotations.md

```diff
@@ -35,6 +35,7 @@ These are all the non-vendor specific annotations inherited from full Modelica t
 These are all the non-vendor specific annotations inherited from full Modelica that are relevant for parameter input and simulation output:
 - `HideResult` — Applied to a parameter or variable, implies the variable should not be included in the simulation output
 - `choices` — Applied to a parameter or variable, can be used to enumerate and tag different values for parameter input
+- `experiment` — Applied to the flat model, provide the default start time, stop time, tolerance, and interval length
```

Comment:
I understand that the goal of "flat model" is to say that it is only for the top-level Base Modelica model (and not for any records or functions), but I couldn't find the name for that entity. Obviously "base model" gives the wrong impression. 
Apart from that it seems ok.

Reply:
Can't we simply express it like this?
```suggestion
- `experiment` — Applied to the `model`, provide the default start time, stop time, tolerance, and interval length
```